vue/sbc-common-components/src/services/token.services.ts

```diff
@@ -77,14 +78,14 @@ class TokenServices {
   private scheduleRefreshToken (refreshEarlyTimeinMilliseconds: number) {
     let refreshTokenExpiresIn = -1
     // check if refresh token is still valid . Or else clear all timers and throw errors
-    if (this.kc && this.kc.timeSkew && this.kc.refreshTokenParsed) {
+    if (this.kc && this.kc.timeSkew !== undefined && this.kc.refreshTokenParsed) {
```

Comment:
I had to think here about why you did this, then realized that a value of 0 is valid.

Reply:
That's correct.